Documentation/concepts/networking/fragmentation.rst

```diff
@@ -0,0 +1,22 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _concepts_fragmentation:
+
+IP fragment handling (beta)
+===========================
+
+By default, Cilium configures the eBPF datapath to perform IP fragment tracking
+to allow protocols that do not support segmentation (such as UDP) to
+transparently transmit large messages over the network. IP fragment tracking
+is implemented in the kernel using an LRU (*Least Recently Used*) map which
```

Comment:
Nit:

```suggestion
is implemented with eBPF using an LRU (*Least Recently Used*) map which
```

Or something like this? Just to make it clear that we don't leave it to the Linux stack?